runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -484,20 +484,12 @@ J9::ARM64::TreeEvaluator::awrtbariEvaluator(TR::Node *node, TR::CodeGenerator *c
       if (translatedNode->getOpCode().isRightShift()) // optional
          translatedNode = translatedNode->getFirstChild();
```

Comment:
The code involving `translatedNode` seems to be dead, here and in all other codegens. Can we remove it?